src/vg-tooltip.js

```diff
@@ -289,12 +289,37 @@
     // the supplemented field option
     var supplementedFieldOption = {};
 
-    // supplement field name with underscore prefixes and suffixes to match the field names in item.datum
-    // for aggregation and timeUnit, this will add prefix "mean_", "yearmonth_"
+    // supplement a user-provided field name with underscore prefixes and suffixes to
+    // match the field names in item.datum
+    // for aggregation, this will add prefix "mean_" etc.
+    // for timeUnit, this will add prefix "yearmonth_" etc.
     // for bin, this will add prefix "bin_" and suffix "_start". Later we will replace "_start" with "_range".
     supplementedFieldOption.field = fieldDef.field ?
       vl.fieldDef.field(fieldDef) : fieldOption.field;
 
+    // If a fieldDef is a (TIMEUNIT)T, we check if the original T is present in the vlSpec.
+    // If only (TIMEUNIT)T is present in vlSpec, we set `removeOriginalTemporalField` to T,
+    // which will cause function removeDuplicateTimeFields() to remove T and only keep (TIMEUNIT)T
+    // in item data.
+    // If both (TIMEUNIT)T and T are in vlSpec, we set `removeOriginalTemporalField` to undefined,
+    // which will leave both T and (TIMEUNIT)T in item data.
+    // Note: user should never have to provide this boolean in options
+    if (fieldDef.type === "temporal" && fieldDef.timeUnit) {
+      // in most cases, if it's a (TIMEUNIT)T, we remove original T
+      var originalTemporalField = fieldDef.field;
+      supplementedFieldOption.removeOriginalTemporalField = originalTemporalField;
```

Comment:
I don't like the fact that `removeOriginalTemporalField` is a string that refer to a field because the variable name gives no clue that this is a string at all.  That said, please file an issue and deal with this minor detail later.  
(File issue & merge -- don't deal with this now.)